specs/fulu/das-core.md

```diff
@@ -118,6 +132,16 @@ def get_custody_groups(node_id: NodeID, custody_group_count: uint64) -> Sequence
     return sorted(custody_groups)
 ```
 
+### `get_max_blobs_per_block`
+
+```python
+def get_max_blobs_per_block(epoch: Epoch) -> uint64:
+    for entry in sorted(BLOB_SCHEDULE, key=lambda e: e["EPOCH"], reverse=True):
+        if epoch >= entry["EPOCH"]:
+            return entry["MAX_BLOBS_PER_BLOCK"]
+    return MAX_BLOBS_PER_BLOCK_ELECTRA
```

Comment:
I do think it's worth simplifying this to just have an entry for electra in the blob schedule, we could even specify the deneb blob limit from the deneb epoch, and assert if we get through the list without a result...

This allows a single data structure for these definitions going forward, and from electra we could remove the use of the old constants, or just leave them dangling and not referenced, as we won't need them...

```
BLOB_SCHEDULE:
     # Electra mainnet fork
  - EPOCH: 364032
    MAX_BLOBS_PER_BLOCK: 6
    # Deneb mainnet fork
  - EPOCH: 269568
    MAX_BLOBS_PER_BLOCK: 3
```